_variable.tf

```diff
@@ -255,6 +255,27 @@ kd+ONf43CmqDSTa3atSFq4sCAwEAAQ==
 EOT
 }
 
+variable "ecr_viewer_auth_api_pub_key" {
+  type        = string
+  description = "The public key used to validate the incoming authenication for the eCR Viewer API."
+  default     = <<EOT
+-----BEGIN PUBLIC KEY-----
```

Comment:
@mcmcgrath13 Will you remove this default? We're going to add a value downstream.

Reply:
should I delete it for both nbs keys?

Reply:
pushed up deleting both - let me know if the original one needs restoring

Reply:
Delete both! Thanks.

Reply:
Ah, we'll need a default so that we aren't required to fill in a prompt or pass something in when Terraform is run.

`default = ""`